recipes/parso/LICENSE.txt

```diff
@@ -0,0 +1,86 @@
+All contributions towards parso are MIT licensed.
```

Comment:
You can delete the `LICENSE.txt` file since it will be pulled from the source distribution.